### DISCUSSION TITLE
Custom repository roles API - Beta Feedback

### DISCUSSION THREAD

**Discussion by hpsin** [community/community#32250] [Enterprise]
We recently shipped a public beta of [new APIs](https://docs.github.com/en/enterprise-cloud@latest/rest/orgs/custom-roles) that allow admins to create and update [custom repository roles](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-peoples-access-to-your-organization-with-roles/managing-custom-repository-roles-for-an-organization).  Custom repository roles are included as part of the Enterprise account, and allow an admin to create least-privilege roles to assign to users and teams within their organization's repositories. 

We're interested in feedback and questions that folks have as they exercise these new APIs.  Thank you!

**Comment by 3kh0** [comment]
Thank you for this new system!
